packages/fuels-signers/src/wallet.rs

```diff
@@ -322,7 +322,7 @@ impl Wallet {
         let script = vec![
             Opcode::gtf(0x10, 0x00, GTFArgs::ScriptData),
             Opcode::ADDI(0x11, 0x10, ContractId::LEN as u16),
-            Opcode::LW(0x13, 0x11, 0),
```

Comment:
Document the change for the LW opcode

Reply:
I don't think there's any need. I only changed the registry and the script remained the same.

Reply:
Oh yes you're right, the documentation was "wrong" before, we didn't explain that we loaded inside 0x13.